File: pset6/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

void mario(int n, int m);

int main(void)
{
    int n = get_int("Heigh: ");
    mario(n, 0);
}

void mario(int n, int m)
{
    if (n == 0)
    {
        return;
    }
    mario(n - 1, m + 1);
    for (int i = 0; i < m; i++)
    {
        printf(" ");
    }
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
    printf("  ");
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: pset6/mario/more/mario.py
from cs50 import get_int

n = 0
while (n <= 0 or n > 8):
    n = get_int("Heigh: ")
space = n - 1
for i in range(n):
    print(" "*space + "#"*(i+1) +'  '+"#"*(i+1))
    space = space - 1
